pycheribuild/projects/cross/qt5.py

```diff
@@ -2,6 +2,8 @@
 # Copyright (c) 2017 Alex Richardson
 # All rights reserved.
 #
+# Copyright (c) 2022 Arm Limited and/or its affiliates <open-source-office@arm.com>
```

Comment:
Is this change really copyrightable (especially once every part of it is rewritten by me...)?

Reply:
I am not a lawyer, but my understanding is broadly that copyrightability depends on jurisdiction, and presumably many other things. In any case, Arm asks us (by default) to update copyright statements on _all_ contributions so we don't need to apply our own personal heuristics, which will inevitably be wrong somewhere.

> especially once every part of it is rewritten by me
Well that makes things much easier, yes :-)